packages/cascara/src/ui/Table/Table.js

```diff
@@ -95,7 +95,7 @@ const Table = ({
     );
   }
 
-  const unwantedActionButtonIndex = dataConfig?.actionButtonIndex;
+  const unwantedActionButtonIndex = dataConfig?.actionButtonMenuIndex;
```

Comment:
Was this a bugfix we we missed?

Reply:
Yes

Reply:
In the future, let's make sure we are creating a ticket for these when we see them so we know what is going on. If it is one line like this we are okay making the ticket and then including the ticket number in the commit message. We do at least need to track that somewhere.